From 2685e50e672748cf3d43ba90e14e895ac890a747 Mon Sep 17 00:00:00 2001
From: Usty <usty@uase.tech>
Date: Thu, 3 Sep 2026 12:04:46 +0000
Subject: [PATCH] chore: bump Android version 1.0.4 (5) -> 1.0.5 (6) for the
 production release with all fixes since production access was granted

---
 android/app/build.gradle | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/android/app/build.gradle b/android/app/build.gradle
index c894c5f..4f984cf 100644
--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -7,8 +7,8 @@ android {
         applicationId "com.uasetechstudio.carstrims"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 5
-        versionName "1.0.4"
+        versionCode 6
+        versionName "1.0.5"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         aaptOptions {
             ignoreAssetsPattern = '!.svn:!.git:!.ds_store:!*.scc:.*:!CVS:!thumbs.db:!picasa.ini:!*~'
-- 
2.43.0

